.gitignore

```diff
@@ -1,6 +1,6 @@
 # Eclipse stuff
-/.classpath
```

Comment:
Those aren't folders, but they reference only the contents inside the root folder of the project which doesn't make sense neither. I didn't noticed that earlier thank you very much for pointing that out.